pom.xml

```diff
@@ -29,21 +29,21 @@
     <module>waggle-dance-boot</module>
     <module>waggle-dance-integration-tests</module>
     <module>waggle-dance</module>
-    <module>waggle-dance-rpm</module>
+<!--    <module>waggle-dance-rpm</module>-->
   </modules>
 
   <properties>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
     <spring-boot.version>2.0.4.RELEASE</spring-boot.version>
-    <hadoop.version>2.7.2</hadoop.version>
+    <hadoop.version>3.1.0</hadoop.version>
     <hamcrest.version>2.2</hamcrest.version>
-    <hive.version>2.3.7</hive.version>
+    <hive.version>3.1.2</hive.version>
     <junit.version>4.13.1</junit.version>
     <mockito.version>3.5.15</mockito.version>
     <dropwizard.metrics.version>3.1.5</dropwizard.metrics.version>
     <aspectj-maven-plugin.version>1.9</aspectj-maven-plugin.version>
     <aspectj.version>1.8.9</aspectj.version>
-    <beeju.version>4.0.0</beeju.version>
+    <beeju.version>5.0.0</beeju.version>
```

Comment:
Please merge the current state of `hive-3.x` into your branch as this change (and some others) have been made there now.